core/src/main/java/me/mykindos/betterpvp/core/gamer/GamerManager.java

```diff
@@ -31,6 +34,10 @@ public Optional<Gamer> getGamerByName(String name) {
         return objects.values().stream().filter(gamer -> gamer.getClient().getName().equalsIgnoreCase(name)).findFirst();
     }
 
+    public List<Gamer> getOnlineOfRank(Rank rank) {
```

Comment:
This isn't really returning online gamers, perhaps just called it getGamersWithRank

Reply:
Yeah I thought so, fixed.